lib/data/demos.dart

```diff
@@ -442,6 +444,38 @@ List<GalleryDemo> materialDemos(GalleryLocalizations localizations) {
       ],
       category: GalleryDemoCategory.material,
     ),
+    GalleryDemo(
+      title: localizations.demoDividerTitle,
+      icon: GalleryIcons.divider,
+      slug: 'divider',
+      subtitle: localizations.demoDividerSubtitle,
+      configurations: [
+        GalleryDemoConfiguration(
+          title: localizations.demoDividerTitle,
+          description: localizations.demoDividerDescription,
+          documentationUrl: '$_docsBaseUrl/material/Divider-class.html',
+          buildRoute: (_) => const DividerDemo(),
+          code: CodeSegments.dividerDemo,
+        ),
+      ],
+      category: GalleryDemoCategory.material,
+    ),
+    GalleryDemo(
+      title: localizations.demoVerticalDividerTitle,
```

Comment:
No need for it to be a separate entry, it can just be added as an additional `GalleryDemoConfiguration` above